src/templates/assets/stylesheets/main/extensions/pymdownx/_details.scss

```diff
@@ -69,6 +69,10 @@
     display: block;
     min-height: px2rem(20px);
     padding-inline-end: px2rem(36px);
+    // Hack: Work around Firefox bug that renders a subpixel outline when
+    // rotating a mask image element.
+    // https://bugzilla.mozilla.org/show_bug.cgi?id=1671784
```

Comment:
Could you minify the link please? I use [t.ly](https://t.ly/) nowadays, as bit.ly became unusable. Please reference the link similar to how we do it in other parts of our code base, e.g.:

https://github.com/squidfunk/mkdocs-material/blob/c0755bf2471a3476d9592de99384abe476c4a645/src/plugins/blog/readtime/__init__.py#L34

Reply:
Oh, and maybe you could also do that here, that slipped my attention:

https://github.com/squidfunk/mkdocs-material/blob/c3683b240f665c2cca87d1cf59e784e0f827b244/src/templates/assets/stylesheets/main/components/_tooltip.scss#L165-L167

Reply:
Done. :slightly_smiling_face: